.travis.yml

```diff
@@ -33,7 +33,7 @@ install:
   - if [[ ${TYPE} == "Integration" ]]; then cp -r archetypes $SMALLTALK_CI_BUILD; fi
   
 script:
-  - if [[ ${TYPE} == "Integration" ]]; then "$SMALLTALK_CI_HOME/run.sh" ".smalltalk.${TYPE}.ston"; fi
+	- if [[ ${JOB} == "test" ]] && [[ ${TYPE} != "System" ]]; then "$SMALLTALK_CI_HOME/run.sh" ".smalltalk.${TYPE}.ston"; fi
```

Comment:
The indentation is not good: travis rejects the file...

Reply:
Okay, I am fixing that